ApplyToBecomeInternal/ApplyToBecome.Data/Services/HttpClientService.cs

```diff
@@ -25,7 +25,17 @@ public HttpClientService(IHttpClientFactory httpClientFactory, ILogger<HttpClien
 			var result = await _client.PostAsync(path, requestPayload);
 
 			return await HandleResponse<TResponse>(result);
-		}	
+		}
+
+		public async Task<ApiResponse<TResponse>> Put<TRequest, TResponse>(string path, TRequest requestBody)
+			where TResponse : class
+		{
+			var requestPayload = JsonContent.Create(requestBody);
+			var x =  await requestPayload.ReadAsStringAsync();
```

Comment:
If this variable is needed could you please change to a readable name, however it looks like this line isn't required as then next one doesn't use 'x' just the request payload

Reply:
Thanks @MartinWheelerMT. I put that in to help debug an issue, it shouldnt be there any more. Will update it now.